src/handlers/Cat.ts

```
@@ -0,0 +1,27 @@
+import { SlashRoute, SlashHandler } from "@seedcord/gateway"
+import { ContainerBuilder } from "discord.js"
```

ESLint

[error] Import component builders from '`@discordjs/builders`'. The discord.js re-export is a separate CJS copy that breaks instanceof and toJSON when nested in a seedcord component.

(`@seedcord/no-djs-builder-import`)

---

[error] Unable to resolve path to module 'discord.js'.

(import-x/no-unresolved)